docs/3_1_model_exchange_math.adoc

```diff
@@ -208,15 +208,15 @@ All discrete-time variables are fixed during this phase and the corresponding di
 ===== Event Mode
 This mode is used to compute new values for all continuous-time variables, as well as for all discrete-time variables that are activated at the current event instant latexmath:[t], given the values of the variables from the previous instant latexmath:[{}^{\bullet}t].
 This is performed by solving algebraic equations consisting of all continuous-time and all active discrete-time equations.
-In FMI 2.0 there is no mechanism that the FMU can provide the information whether a discrete-time variable is active or is not active (is not computed) at an event instant.
+In FMI, there is no mechanism that the FMU can provide the information whether a discrete-time variable is active or is not active (is not computed) at an event instant.
```

Comment:
Is this true for clocks and clocked variables? But this is not the intention of this PR, so I create a new ticket https://github.com/modelica/fmi-standard/issues/1045

Reply:
@chrbertsch : I tried to fix this in a later commit (117a288) to this branch - can you check and if you approve, merge this PR?